client/app/styles/onboarding.styl

```diff
@@ -240,3 +242,20 @@ footer
 .my-accounts-container
     flex-grow 1
     border-width 0
+
+.my-accounts-footer
+    font-size 16px
+    height  (48 / 16)em
```

Comment:
Even we should a global variable here to store this `16px` as font-size, we could use a local variable here for this stylus part. What's your opinion about that?